logic/peer.js

```diff
@@ -25,7 +38,9 @@ Peer.prototype.nullable = [
 	'version',
 	'dappid',
 	'broadhash',
-	'height'
+	'height',
+	'clock',
+	'updated'
 ];
 
 // Public methods
```

Comment:
the accept function works fine (accepts) peer even without port and ip. Is it ok?